tools/cliupdate.php

```diff
@@ -359,17 +372,19 @@ function displayWarning($msg, $red=false) {
       update91to92();
       break;
 
-   case GLPI_VERSION :
+   case GLPI_VERSION:
+   case GLPI_SCHEMA_VERSION:
+   case GLPI_PREVER:
```

Comment:
Yep, this should be the case, if you use the `--force` option. The swith here is only to prevent using the `default` case, and get an error.